featurebyte/api/groupby.py

```diff
@@ -468,7 +468,7 @@ def forward_aggregate(
         method: Union[AggFunc, str],
         window: str,
         target_name: str,
-        fill_value: OptionalScalar = None,
+        fill_value: OptionalScalar,
```

Comment:
The old behaviour was that for `count` aggregation methods, the `fill_value` is automatically set as 0. Should we restore that behaviour? It seems slightly more convenient.